salad/operator/config/testnet_config.json

```diff
@@ -1,11 +1,13 @@
 {
- "CONTRACT_DISCOVERY_ADDRESS": "http://contract:8081",
- "ETH_NODE_ADDRESS": "contract",
- "ETH_NODE_PORT": "9545",
- "ENG_NODE_ADDRESS": "worker",
- "ENG_NODE_PORT": "3346",
- "MONGO_URL": "mongodb://mongo:27017",
- "DB_NAME": "salad",
- "SECRET_CONTRACT_BUILD_FOLDER": "..",
- "KEY_MANAGEMENT_DISCOVERY": "THIS IS REDUNDANT FOR SALAD OPERATOR, REMOVE IN THE FUTURE"
-}
+    "CONTRACT_DISCOVERY_ADDRESS": "http://contract:8081",
+    "ETH_NODE_ADDRESS": "parity.enigma.co",
+    "ETH_NODE_PORT": "8545",
+    "ENG_NODE_ADDRESS": "worker",
+    "ENG_NODE_PORT": "3346",
+    "MONGO_URL": "mongodb://mongo:27017",
+    "DB_NAME": "salad",
+    "SECRET_CONTRACT_BUILD_FOLDER": "..",
+    "SALAD_SMART_CONTRACT_ADDRESS": "0x3ad332C3098C04bc60C23c7C469bBF19590C7848",
+    "SALAD_SECRET_CONTRACT_ADDRESS": "0x6bb065c82b68327ed064c4974211ede0f568517db84dc95aa77a274a9c2cfb29",
+    "KEY_MANAGEMENT_DISCOVERY": "THIS IS REDUNDANT FOR SALAD OPERATOR, REMOVE IN THE FUTURE"
```

Comment:
Don't need this anymore

Reply:
what exactly is not needed? only KEY_MANAGEMENT_DISCOVERY?